src/visualization/src/layer.h

```diff
@@ -0,0 +1,41 @@
+#ifndef LAYER_H_
+#define LAYER_H_
+
+#include "javascript_caller.h"
+
+#include <memory>
+
+#include "include/cef_app.h"
+#include "include/cef_render_process_handler.h"
+#include "include/cef_process_message.h"
+
+class Layer : public CefApp, public CefBrowserProcessHandler, public CefRenderProcessHandler {
```

Comment:
What does "Layer" mean in this context - why is it called Layer?

Reply:
Layer is the CEF Browser Layer